src/hooks/level_cell.cpp

```diff
@@ -26,12 +34,30 @@ void GlobedLevelCell::updatePlayerCount(int count, bool inLists) {
             .parent(this)
             .id("player-count-label"_spr)
             .store(m_fields->playerCountLabel);
+
+        Build<CCSprite>::createSpriteName("icon-person.png"_spr)
+                .scale(iconScale)
+                .anchorPoint(1.f, 0.5f)
+                .pos(xPos, yPos)
+                .parent(this)
+                .id("player-count-icon"_spr)
+                .store(m_fields->playerCountIcon);
+
+        m_fields->playerCountIcon->setVisible(false); // :D
     }
 
     if (count == 0) {
         m_fields->playerCountLabel->setVisible(false);
     } else {
         m_fields->playerCountLabel->setVisible(true);
-        m_fields->playerCountLabel->setString(fmt::format("{} {}", count, count == 1 ? "player" : "players").c_str());
+
+        if (settings.globed.compressedPlayerCount) {
+            m_fields->playerCountIcon->setVisible(true);
+            m_fields->playerCountLabel->setPositionX(m_fields->playerCountLabel->getPositionX() - 13);
```

Comment:
wouldn't this push the icon slightly more and more to the left whenever the player count updates? or am I misunderstanding

Reply:
it resets the position of the label on update iirc, i'll check in a sec